src/lib/stores/i18n.ts

```diff
@@ -220,6 +220,11 @@ i18next.init({
               dayWithCount_one: "{{count}} day",
               dayWithCount_other: "{{count}} days",
             },
+            startOfWeek: {
+              name: "Start Week On",
```

Comment:
```suggestion
              name: "Start week on",
```

UI uses sentence casing.